src/server/util/spec_resolvers.cc

```diff
@@ -102,12 +102,6 @@ DEFINE_bool(sync_crds, false, "Synchronize CRDs when persisting objects");
 // IO: spill and migration
 DEFINE_bool(compression, true, "Compress before migration or spilling");
```

Comment:
So it seems the binary may need to take additional command-line flags, so the client initialization may be add odds with this.

Reply:
This is on the server side. Client SDK only have two flags right now.

Reply:
Got it.